### ISSUE TITLE
i18n support for UX improvement

### ISSUE THREAD

**Issue by Bluesy1** [Bluesy1/CharB0T#158]
Using [fluent](https://projectfluent.org/fluent/guide/)

- [x] Convert all localize-able strings to load from the corresponding `.ftl` files to allow for i18n
- [x] Setup system to allow people to submit translation suggestions
- [x] Setup i18n in dpy (blocked on i18n being merged into stable (currently separate branch).

**Comment by Bluesy1** [comment]
Closed by #157 in merge e38270c3bf1290e14364efce454f652f7bad06f9
